packages/init/bin/d2-init

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env node
+console.log('creating sds ...')
```

Comment:
this is a placeholder for a new project that would work as an alias to `app scripts` allowing consumers to scaffold a new app using the more standard `npx create @dhis2`